lib/internal/bootstrap_node.js

```diff
@@ -208,7 +208,6 @@
 
   function setupGlobalVariables() {
     global.process = process;
-    global.global = global;
```

Comment:
This is now done in https://github.com/nodejs/node/pull/5881/files#diff-cc63475f116262e9cbea02374e344c01L211